src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -666,19 +667,27 @@
          * @param {Object[]|null} uhIdentifiers
          */
         $scope.addMembers = (listName, uhIdentifiers) => {
+            // Prevent multiple asynchronous calls of addMembers
+            if ($scope.addingMembers === true) {
+                return;
+            }
```

Comment:
When naming booleans, we tend to prefix it with "is" so that it is more clean to read when doing the following:
```
if ($scope.isAddingMembers) {
    return;
}
```
That way, you do not need to specifically check for `=== true`. This can be more dangerous in Javascript without any typing restrictions like Typescript has, but we can definitely trust a variable to be a boolean with that naming convention.

Reply:
Should I also do this for my previous ticket of the same issue? https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/pull/998

Reply:
Yeah good catch, you can include the change in this ticket. I guess you could keep the variable name $scope.loadingOwners the same because we already have $scope.loading everywhere instead of $scope.isLoading.